src/frontends/pytorch/src/utils.cpp

```diff
@@ -101,6 +101,33 @@ Output<Node> reshape_kernel_for_group(const NodeContext& context,
     return make_shared<opset8::Reshape>(kernel, new_kernel_shape, false);
 }
 
+std::shared_ptr<Node> get_im2col_indices_along_dim(NodeContext& context,
```

Comment:
Will you use this function in other translators? If no, it would be better to have it inside the `im2col` translator.

Reply:
I suppose, it will be reused for unfold op, but for now I can hide

Reply:
moved